src/modules/enrollment/components/dashboardPages/EnrollmentCustomCaseNotesPage.tsx

```diff
@@ -0,0 +1,158 @@
+import AddIcon from '@mui/icons-material/Add';
+import { Box, Button } from '@mui/material';
+import { useCallback, useState } from 'react';
+
+import { ColumnDef } from '@/components/elements/table/types';
+import TitleCard from '@/components/elements/TitleCard';
+import { useEnrollmentDashboardContext } from '@/components/pages/EnrollmentDashboard';
+import NotFound from '@/components/pages/NotFound';
+import DeleteMutationButton from '@/modules/dataFetching/components/DeleteMutationButton';
+import GenericTableWithData from '@/modules/dataFetching/components/GenericTableWithData';
+import { useFormDialog } from '@/modules/form/hooks/useFormDialog';
+import useViewDialog from '@/modules/form/hooks/useViewDialog';
+import { parseAndFormatDateTime } from '@/modules/hmis/hmisUtil';
+import { cache } from '@/providers/apolloClient';
+import {
+  CustomCaseNoteFieldsFragment,
+  DeleteCustomCaseNoteDocument,
+  DeleteCustomCaseNoteMutation,
+  DeleteCustomCaseNoteMutationVariables,
+  FormRole,
+  GetEnrollmentCustomCaseNotesDocument,
+  GetEnrollmentCustomCaseNotesQuery,
+  GetEnrollmentCustomCaseNotesQueryVariables,
+} from '@/types/gqlTypes';
+
+const columns: ColumnDef<CustomCaseNoteFieldsFragment>[] = [
+  {
+    header: 'Content',
+    render: ({ content }) => (
+      <Box sx={{ whiteSpace: 'pre-wrap' }}>{content}</Box>
```

Comment:
Can we ellipsize this if its really long? like over 800 chars maybe?

Reply:
Probably use line css line-clamp, that seems to work in the evergreen browsers